tensor/datafs/metadata.go

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package datafs
+
+import (
+	"cogentcore.org/core/base/errors"
+	"cogentcore.org/core/base/fsx"
+	"cogentcore.org/core/base/metadata"
+	"cogentcore.org/core/plot/plotcore"
+	"cogentcore.org/core/tensor/table"
+)
+
+// This file provides standardized metadata options for frequent
+// use cases, using codified key names to eliminate typos.
```

Comment:
I really don't like all of these specialized helper functions that dump all of the `emergent`-style garbage into core; could you at least move these to a different package?

Reply:
this is core functionality used in browser!